types/react/index.d.ts

```diff
@@ -785,7 +785,7 @@ declare namespace React {
         // Pick would not be sufficient for this. We'd like to avoid unnecessary mapping and need a distributive conditional to support unions.
         // see: https://www.typescriptlang.org/docs/handbook/2/conditional-types.html#distributive-conditional-types
         // https://github.com/Microsoft/TypeScript/issues/28339
-        P extends any ? ('ref' extends keyof P ? Pick<P, Exclude<keyof P, 'ref'>> : P) : P;
+        P extends any ? ('ref' extends keyof P ? Omit<P, 'ref'> : P) : P;
```

Comment:
(see prior discussion [here](https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/64163#discussioncomment-4831503)) as far as I can tell this is logically equivalent.

Reply:
This should be equivalent since it's the actual definition of `Omit`

```ts
type Omit<T, K extends keyof any> = Pick<T, Exclude<keyof T, K>>;
Omit<P, 'ref'> = Pick<P, Exclude<keyof P, 'ref'>;
```